lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -393,7 +393,13 @@ def insert_at_position(position)
         end
 
         def update_positions
-          old_position = send("#{position_column}_was") || bottom_position_in_list + 1
+          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
+            old_position = send("#{position_column}_before_last_save")
+          else
+            old_position = send("#{position_column}_was")
```

Comment:
Extracted that block to a method. It's clean and leaves original method basically intact.